tavern_pathfinder/src/lib.rs

```diff
@@ -1,7 +1,622 @@
+extern crate serde;
+extern crate uuid;
+
 #[cfg(test)]
 mod tests {
     #[test]
     fn it_works() {
         assert_eq!(2 + 2, 4);
     }
 }
+pub mod types {
+    use serde::{Serialize, Deserialize};
+
+    #[derive(Serialize,Deserialize)]
```

Comment:
We'd have to look at whether this would work with the database and how we'll use the library, but reminder that rust enums can contain data. So we could, for example, write a value of `Attribute::Strength(15)`. Not sure if this is useful, but may be worth looking into, to leverage the features of the language.

Reply:
I could see it being useful. My approach here was to make it as easy as possible to go from the SQL responses to analogous Rust objects, and so I didn't plan on adding additional complexity - but if it makes it easier on our end in the long run, I'm down for implementing data-containing enums.

Reply:
I would rather complexity in translating the objects than complexity in using them, if that makes sense. We're going to be writing custom code to interact with the database anyway, might as well make use of Rust's features too.